app/src/pages/_app.tsx

```diff
@@ -62,15 +110,30 @@ function App({ Component, pageProps }: AppProps) {
                   backgroundColor: theme.colors[theme.primaryColor][shade],
                   display: 'flex',
                   alignItems: 'center',
-                  paddingLeft: '18px'
+                  justifyContent: 'space-between'
                 };
               }}
             >
-              <Link href="/">
-                <Text c="white" weight="bold">
-                  Measure Repository
-                </Text>
-              </Link>
+              <div className={classes.brandContainer}>
+                <Link href="/">
+                  <Group align="center" spacing="xs">
+                    <Center>
+                      <Database className={classes.dbIcon} />
+                    </Center>
+                    <Text c="white" weight="bold">
+                      Measure Repository
+                    </Text>
+                  </Group>
+                </Link>
+              </div>
+              <Group position="center" className={classes.navGroup} spacing="xl">
+                <Link href="/">
+                  <Text c={router.pathname === '/' ? 'orange.3' : 'gray.4'}>Repository</Text>
```

Comment:
![image](https://github.com/projecttacoma/measure-repository/assets/30158156/955c2506-1a4c-406d-ba9e-d58f451487cf)

Do we want "Repository" to be orange when we are browsing the Measure or Library artifacts that are on the mrs?

Reply:
I went back and forth on this, but yeah I think prob we should just keep repository highlighted when not doing authoring stuff. Maybe it will change in the future